utils.sh

```diff
@@ -79,7 +79,7 @@ function provide-parity-tools {
 
     export PATH=~/.cargo/bin:$PATH
 
-    cargo install --git https://github.com/paritytech/cargo-contract.git cargo-contract --force
+    cargo install cargo-contract
```

Comment:
We should use --force here to always get the latest version. 
I'm not sure about the release policy of cargo-contract, but we want the latest master here, not the latest released version. So if not every commit to master triggers a release, I'd leave it to the old `cargo install` command

Reply:
The force is really annoying when developing as it reinstalls it every time when executing `build.sh`. We can remove it here altogether and just let the CI script install the latest master as that is not excuted locally.